test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/ManagedDependenciesEndToEndTests.cs

```diff
@@ -14,7 +15,9 @@ public async Task ManagedDependenciesAreAvailable()
         {
             var actualResponseMessage =
                 await Utilities.InvokeHttpTrigger("UsingManagedDependencies", string.Empty, HttpStatusCode.OK);
-            Assert.Contains(@"\ManagedDependencies\Az\", actualResponseMessage);
+
+            var expectedAzModulePathPart = Path.DirectorySeparatorChar + Path.Combine("ManagedDependencies", "Az") + Path.DirectorySeparatorChar;
```

Comment:
Do you really need the DirectorySeparatorChars here? Do they provide value?

Reply:
Definitely. I want to make sure the module is picked up from the intended location, not something coincidentally similar. For example, I don't want this assert to match `C:\NoIdeaWhyManagedDependencies\AzIsNot\Supposed\To\Be\Here\But\It\Is\Az\1.0`.

You can argue this coincidence is very unlikely to happen, and I will agree with this. But I do have a true and funny story about not being able to install a certain product a couple of months ago, just because the installer was looking for a certain library by using **lib\Something** pattern in the path, found it at C:\Users\anato**lib\Something**EntirelyUnrelated\..., and messed everything up.

Reply:
Sounds good :) just wanted to check.

Reply:
The last one gets you really strange .NET behavior 😅

```
[io.path]::Combine("banana","squash")
[io.path]::Combine("/", "banana","squash")
[io.path]::Combine("/", "banana","squash", "/")
```

(I used “/“ because I’m on a mac for convenience)

Reply:
LGTM

Reply:
> The last one gets you really strange .NET behavior 😅

Yes, this is exactly why I decided to add them separately.